.storybook/guides/deprecation.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Guides/Deprecation workflow"/>
+
+# Deprecating a component
+
+When a component is deprecated, it is no longer recommended for use. Though the component will be removed from the codebase and will no longer receive feature updates, we will apply bug fixes to the package as necessary. We will also provide a migration path to a recommended alternative.
+
+## Deprecation process
+
+**Deprecation of a component is typically initiated and communicated by the design team.** They will indicate that a component is going to be deprecated and identify reccomended alternatives and proposed language around migrating away from that component's use.
+
+### Developer responsibilities
+
+#### Due diligence
+
+Before starting a deprecation, connect with the Spectrum Web Components team to align on the timing of the deprecation. Make sure downstream users are aware that the component is being deprecated and have a migration path to the recommended alternative. Give downstream consumers time to express any concerns or blockers.
+
+#### Storybook
+
+- [ ] Create a new folder with the component name in `.storybook/deprecated`, i.e., `.storybook/deprecated/quickaction`.
+- [ ] Copy `*.stories.js` and `metadata/*.yml` into the new folder (directly, not nested inside subfolders).
+- [ ] Open up and edit the title of the `*.stories.js` default export to be prefixed with the `Deprecated` category, i.e., `title: "Deprecated/Quick actions"`.
+- [ ] Update any local references to point to the package name instead, i.e.,<br/>_Original_:<br/>`import { Template } from "./template";`<br/><br/>_Updated to_:<br/>`import { Template } from "@spectrum-css/quickaction/stories/template.js";`.
+- [ ] In the parameters section of the `*.stories.js`, add `chromatic: { disable: true },` to ensure it no longer runs regression tests.
```

Comment:
Oh! Also here we might want to call out updating the status to "deprecated"

Reply:
Oh!  I totally forgot that step. Yes yes!

Reply:
I also just pushed up a revert of the checkbox syntax because it's not supported in storybook's MDX flavor.